### PR TITLE
[EIP1-8858] update messaging yaml to include up to date document types

### PR DESCRIPTION
**Pull request by TomosSlater**
Add document types used in OE API

### CODE REVIEW

**Review by ollylarkin-softwire — COMMENTED** [review]
Could we bump the version number?

**Review by ollylarkin-softwire — APPROVED** [review]

File: src/main/resources/openapi/NotificationsAPIs.yaml [hunk 1]
```diff
@@ -1833,6 +1833,7 @@ components:
         - duplicate-document
         - invalid-document-country
         - applicant-details-not-clear
+        - parent-guardian-not-clear
```
[the remark this answers is not in the document]
Comment (TomosSlater) [reply src/main/resources/openapi/NotificationsAPIs.yaml:1836]: Have added these, pending some message copy/translations from Damien

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -19,6 +19,7 @@ templates.document-rejection.rejection-reasons.invalid-document-type=The documen
 templates.document-rejection.rejection-reasons.duplicate-document=This was a duplicate of another document that you have provided
 templates.document-rejection.rejection-reasons.invalid-document-country=We are not able to accept documents from this country for the purposes of checking your identity
 templates.document-rejection.rejection-reasons.applicant-details-not-clear=The document needs to clearly show your name
+templates.document-rejection.rejection-reasons.parent-guardian-not-clear=The document needs to clearly show your parent or guardian details
```
Comment (TomosSlater) [inline src/main/resources/messages.properties:22]: final copy pending input from Damien

File: src/main/resources/messages_cy.properties [hunk 3]
```diff
@@ -18,6 +18,7 @@ templates.document-rejection.rejection-reasons.duplicate-document=Roedd hwn yn g
 templates.document-rejection.rejection-reasons.invalid-document-country=Ni allwn dderbyn dogfennau o'r wlad hon at ddibenion gwirio pwy ydych
 templates.document-rejection.rejection-reasons.applicant-details-not-clear=Mae angen i'r ddogfen ddangos eich enw'n glir
 templates.document-rejection.rejection-reasons.details-on-document-do-not-match=Nid yw'r wybodaeth a ddarperir ar y ddogfen yn cyfateb i'r wybodaeth o'ch cais
+templates.document-rejection.rejection-reasons.parent-guardian-not-clear=Mae angen i'r ddogfen ddangos yn glir fanylion eich rhiant new warcheidwad
```
Comment (TomosSlater) [inline src/main/resources/messages_cy.properties:21]: final Welsh copy pending input from Damien

File: src/main/resources/openapi/NotificationsAPIs.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (TomosSlater) [reply src/main/resources/openapi/NotificationsAPIs.yaml:1836]: We're still waiting on the Welsh translation, but don't want this to hold up testing etc, so I've made a separate ticket to track the Welsh translation (and put in a sensible translation for now):

https://dluhcdigital.atlassian.net/browse/EIP1-9224

[the remark this answers is not in the document]
Comment (emisto3) [reply src/main/resources/openapi/NotificationsAPIs.yaml:1836]: Amazing thanks!

[the remark this answers is not in the document]
Comment (emisto3) [reply src/main/resources/openapi/NotificationsAPIs.yaml:1836]: Looks like this was done in a separate PR 🎉

**Review by emisto3 — APPROVED** [review]
Looks good!

**Pull request merged**